styleGuides/apiStyleGuide.md

```diff
@@ -0,0 +1,32 @@
+
+# Api Style Guide
+
+## Paths
+
+API routes must start with the `/api/` path segment, and should be followed by the plugin id if applicable:
+
+*Right:* `/api/marvel/v1/nodes`
+*Wrong:* `/marvel/api/v1/nodes`
```

Comment:
Let's ditch the version in our example since the very next section discourages versioning.